server/views/components/goal-summary-card/_goalSummaryCardContent-goalInStatus-COMPLETED.njk

```diff
@@ -1,19 +1,30 @@
 <div class="govuk-summary-card__content">
   {% include './_goalSteps.njk' %}
 
-  {% if params.goal.notesByType.GOAL.length > 0 %}
-    {% set goalNote = params.goal.notesByType.GOAL[0] %}
-    <details class="govuk-details app-notes-expander" data-qa="overview-notes-expander">
-      <summary class="govuk-details__summary">
-            <span class="govuk-details__summary-text">
+  {% if params.goal.status === 'COMPLETED' %}
+      {% if params.goal.notesByType.GOAL_COMPLETION.length > 0 %}
+        {% set goalCompletionNote = params.goal.notesByType.GOAL_COMPLETION[0] %}
+        <p class="govuk-heading-s">Goal completion note</p>
+        <p class="govuk-body">{{ goalCompletionNote.content }}</p>
+      {% endif %}
+    {% endif %}
+    {% if params.goal.notesByType.GOAL.length > 0 %}
+      {% set goalNote = params.goal.notesByType.GOAL[0] %}
+      <details class="govuk-details app-notes-expander" data-qa="overview-notes-expander">
+        <summary class="govuk-details__summary">
+          <span class="govuk-details__summary-text">
+            {% if params.goal.status === 'COMPLETED' %}
```

Comment:
As above, this if check will always be true, because this is the template specifically for Completed goals.
Therefore it will always show the text "View previous note" ....

Do we have a requirement for this? I didnt see it in the ticket Though to be fair the ticket and screenshots dont show anything about the notes expander and what text it should show. I think we should check this with Jess in 3As today

Reply:
My understanding from the ticket was 'View previous note' and the accompanying expanded note content should appear if there's a previous goal note (that isn't a completed goal note). But will make a note to check at 3as.